vendor/wagn

```diff
@@ -1 +1 @@
-Subproject commit bee8b896f05495d9eb741b048bedc663c911ba3e
+Subproject commit 03cb9f7cc83831c3c0b69e5032fcbe17067a7e28
```

Comment:
in other words, we want it so that the *only* change to master is claim.rb.  I know that didn't work before, but I think it will now, thanks to the semaphore change.